felt/src/lib_bigint_felt.rs

```diff
@@ -19,6 +19,9 @@ use core::{
 #[cfg(all(not(feature = "std"), feature = "alloc"))]
 use alloc::{string::String, vec::Vec};
 
+#[cfg(feature = "fuzzing")]
```

Comment:
These should be changed to
```rust
#[cfg(feature = "arbitrary")]
```

Reply:
Resolved: https://github.com/lambdaclass/cairo-vm/pull/1306/commits/51e5ff5405ae1e520bc13b5057efc8c1df9c30ad